src/google/protobuf/descriptor.h

```diff
@@ -745,6 +745,9 @@ class PROTOBUF_EXPORT FieldDescriptor {
   // nullptr.
   const OneofDescriptor* containing_oneof() const;
 
+  // Is the field in a real one-of (as opposed to a synthetic oneof)?
+  bool is_in_real_oneof() const;
```

Comment:
You should now be able to call `real_containing_oneof()` (this function may not have existed when you started this change). So `descriptor.h` shouldn't need to change.

Reply:
Great - very happy to revert this :)